subprojects/core/src/main/java/org/gradle/api/internal/plugins/ImperativeOnlyPluginApplicator.java

```diff
@@ -44,4 +44,8 @@ public void applyImperativeRulesHybrid(@Nullable String pluginId, Plugin<?> plug
         applyRules(pluginId, plugin.getClass());
     }
 
+    @Override
+    public String toString() {
+        return target.toString();
+    }
```

Comment:
Searching the sources, we have only one other thing named `...Target`, which is  `ScriptTarget`. It also wraps the actual target object and then provides means to couple target and script (`attachScript()`). So there is some analogy.

Sounds like a reasonable rename. I would do it as a separate commit after this PR was merged though. If we rename it, we should open an issue to have a proper record of the name change.